src/components/EditKeyShortcutsWrapper/EditKeyShortcutsWrapper.test.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+
+import '../../../test/jest/__mock__';
+
+import { waitFor } from '@testing-library/react';
+
+import {
+  renderWithReduxForm,
+  translationsProperties,
+  saveShortcut,
+} from '../../../test/jest/helpers';
+
+import { EditKeyShortcutsWrapper } from './EditKeyShortcutsWrapper';
+
+const {
+  CommandList,
+  defaultKeyboardShortcuts,
+} = require('@folio/stripes/components');
+
+const mockOnSubmitProp = jest.fn(async () => 'test value');
+
+jest.mock('@folio/stripes/smart-components', () => ({ ...jest.requireActual('@folio/stripes/smart-components') }), { virtual: true });
+
+const renderEditKeyShortcutsWrapper = ({ onSubmit }) => {
+  const childElement = <span>Child element</span>;
+
+  const component = () => (
+    <CommandList commands={defaultKeyboardShortcuts}>
+      <EditKeyShortcutsWrapper
+        onSubmit={onSubmit}
+      >
+        {childElement}
+      </EditKeyShortcutsWrapper>
+    </CommandList>
+  );
+
+  return renderWithIntl(renderWithReduxForm(component), translationsProperties);
```

Comment:
Can we render this with the final-form? Because we are getting rid of the redux form currently

Reply:
Noted